ui/app/AppLayouts/Profile/panels/ProfileShowcaseCommunitiesPanel.qml

```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.15
+
+import utils 1.0
+
+import AppLayouts.Profile.controls 1.0
+
+ProfileShowcasePanel {
+    id: root
+
+    settingsKey: "communities"
+    keyRole: "id"
+    roleNames: ["id", "name", "amISectionAdmin", "image", "color"]
+    filterFunc: (modelData) => modelData.joined && !showcaseModel.hasItem(modelData.id)
+    hiddenPlaceholderBanner: qsTr("Communities here will show on your profile")
+    showcasePlaceholderBanner: qsTr("Communities here will be hidden from your profile")
+
+    draggableDelegateComponent: CommunityShowcaseDelegate {
+        Drag.keys: ["x-status-draggable-showcase-item-hidden"]
+        showcaseObj: modelData
+        dragParent: dragParentData
+        visualIndex: visualIndexData
+        onShowcaseVisibilityRequested: {
+            var tmpObj = Object()
+            root.roleNames.forEach(role => tmpObj[role] = showcaseObj[role])
+            tmpObj.showcaseVisibility = value
```

Comment:
Check if the visibility actually changed. 

Currently we have this scenario that is not working properly:
1. All items are Hidden
2. Open the visibility menu and click on `No one` again
3. The item is moved to `In showcase`

Another option would be to avoid emmitting `showcaseVisibilityRequested` `onClick` and use `onCheckedButtonChanged`. Something like this:
```
//ButtonGroup component
onCheckedButtonChanged: function() {
    root.showcaseVisibilityRequested((checkedButton as ShowcaseVisibilityAction).showcaseVisibility)
}
```

Reply:
We have the same behaviour on all showcase panels

Reply:
Ah, that was the issue that Noelia pointed out and I couldn't reproduce, thanks :) Will fix

Reply:
Fixed